packages/css/src/components/breadcrumb/breadcrumb.scss

```diff
@@ -55,10 +50,6 @@
   text-decoration-thickness: var(--amsterdam-breadcrumb-item-link-text-decoration-thickness);
   text-underline-offset: var(--amsterdam-breadcrumb-item-link-text-underline-offset);
 
-  &::first-letter {
-    text-transform: capitalize;
```

Comment:
Don't we want to enforce a capital first letter?

Reply:
No, I think we shouldn’t tamper with text people write.